local-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2'
+services:
+    mongodb:
+        image: autopilotpattern/mongodb:latest
+        restart: always
+        mem_limit: 512m
+        build: .
+        env_file: _env
+        environment:
+            - CONTAINERPILOT=file:///etc/containerpilot.json
+            - CONSUL=consul
+        ports:
+            - 27017
+
+    consul:
+        image: consul:v0.6.4
+        command: agent -dev -ui -client=0.0.0.0
+        restart: always
+        mem_limit: 128m
+        ports:
+            - 8500
```

Comment:
Usability suggestion for local testing: if this is `8500:8500` the user doesn't have to do a port lookup to pull up the web UI.